.gitignore

```diff
@@ -44,12 +45,6 @@ deps/dpdk*
 # OSX
 .DS_Store
 
-# Protobuf
```

Comment:
Hm, I didn't try building the sample_plugin using `--plugin`, maybe I should make sure that still works too.